templates/amt/sections/avslag-med-begrunnelse-og-vurdering.hbs

```diff
@@ -0,0 +1,23 @@
+{{#each endringer}}
+    <section>
+        <h2>{{this.tittel}}</h2>
+        {{#if this.aarsak}}
+            <p>Årsak: {{this.aarsak}}</p>
+        {{/if}}
+        {{#if this.begrunnelseFraNav}}
+         <div class="forslag">
+            <p class="fritekst">NAVs begrunnelse: {{this.begrunnelseFraNav}}</p>
+           </div>
+        {{/if}}
+
+        <h3>Vurdering fra arrangor</h3>
```

Comment:
Noen tanker her: 

1. Vurderinger er vel bare noe som gruppe-amo har, er denne seksjonen kun for gruppe-amo?
2. Jeg vet ikke om vurderinger er påkrevd. Jeg tror det kan hende at selv om de får et avslag så har de ingen vurderinger.
3. En vurdering er ikke et forslag, men en egen datastruktur.

Reply:
1. Vurderinger kan også være på fag- og yrkesopplæring, men ikke på jobbklubb.